guide_to_writing_techreports.tex

```diff
@@ -1260,75 +1268,82 @@ \subsection{The reference list}
 same the sources are quoted alphabetically according to titles and
 distinguished by alphabetical letters (like 1988a, 1988b).
 
-All the authors are mentioned:  \emph{et al}  is never used in the
+All the authors are mentioned;  \emph{et al}  is never used in the
 reference list.
 
 Details of the publications are given in the following sequence:
 author, year of publication and title; after that it depends on
 whether the publication is a book, a journal or other work.  Most of
 the options are indicated here as examples.  
-Note that these groupings (Book, Paper, etc) should not appear in the reference list.
+Note that these groupings (Book, Paper, \emph{etc}) should not appear 
```

Comment:
Same as above. Also note we should reverse my bad habit of using `\emph` for italics. If we want italics we should use `\textit`